.circleci/config.yml

```diff
@@ -487,6 +503,51 @@ jobs:
           replace: $SNAPSHOT_VERSION
       - notify_github:
           data: "$(jq -Rcs '{ body: . }' scripts/snapshot-comment.md)"
+  build_check_android:
+    parameters:
+      working_directory:
+        type: string
+      android_path:
+        type: string
+    executor:
+      name: android/android-machine
+      tag: '2024.01.1'
+    steps:
+      - advanced-checkout/shallow-checkout
+      - build_and_pack_sdk:
+          working_directory: << parameters.working_directory >>
+      - run:
+          name: Build Android
+          command: |
+            cd << parameters.android_path >>
+            ./gradlew assembleRelease
+
+  build_check_ios:
+    parameters:
+      working_directory:
+        type: string
+      ios_path:
+        type: string
+    macos:
+      xcode: 13.4.1
+    resource_class: macos.m1.medium.gen1
+    steps:
+      - advanced-checkout/shallow-checkout
+      - build_and_pack_sdk:
+          working_directory: << parameters.working_directory >>
+      - install_pods:
+          working_directory: << parameters.ios_path >>
+      - run:
+          name: Build iOS
+          command: |
+            cd << parameters.ios_path >>
+            yarn react-native bundle --platform ios --dev false --entry-file index.js --bundle-output ios/main.jsbundle --assets-dest ios
+            rm -rf ~/Library/Developer/Xcode/DerivedData/*
```

Comment:
Why do we need to remove the `DerivedData` on CI? 👀 Is that due to some flakiness?

Reply:
yes the ci was randomly keeping failing with the drived data